.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -11,8 +11,9 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-name: \U0001F680 Feature Request
+name: Feature Request
 description: Submit a proposal/request for a new feature
+title: "[Feature request]: "
 projects: [ "SAFEHR-data/13" ]
```

Comment:
can remove this line as well

Reply:
sorted out here [9061750](https://github.com/SAFEHR-data/PIXL/pull/514/commits/90617507131d1366017b075312dac717425a6e68)